ReactWindows/ReactNative.Shared.Tests/Modules/WebSocket/WebSocketModuleTests.cs

```diff
@@ -45,29 +47,37 @@ public void WebSocketModule_OpenClosedEvent()
             finally
             {
                 module.close(1000, "None", 1);
-                Assert.IsTrue(waitHandle.WaitOne());
+                Assert.IsTrue(waitHandle.WaitOne(5000));
+                waitHandle.Dispose(); //??
```

Comment:
>//?? [](start = 38, length = 4)

I assume these EventWaitHandles will call dispose in the finalizer, but I do agree that we should make sure to dispose these kinds of resources wherever possible.